### PR TITLE
Add awesome open minds and Tchamba (pip package)

### PR DESCRIPTION
**Pull request by Fcmam5**

### CODE REVIEW

**Review by IOAyman — CHANGES_REQUESTED** [review]
Please rebase on `upstream/master`

File: README.md [hunk 1]
```diff
@@ -28,12 +28,15 @@ A curated list of awesome things and projects built by Algerian developpers.
...
   - [ArduinoSpritzCipher](https://github.com/abderraouf-adjal/ArduinoSpritzCipher) - Spritz library for Arduino, CSPRNG, cryptographic hash and MAC functions, symmetric-key data encryption, and general-purpose functions
+- Python
+  - [Tchamba.random](https://github.com/Fcmam5/tchamba) package of functions generating random data (colors, names, jokes, letters..)
 - Other languages
   - [Arabic Stremmer](https://github.com/assem-ch/arabicstemmer) - Arabic Light Stemmer is a snowball-based stemming algorithm for Arabic
 
 ## Other
 - [Tamazight Keyboard Layout](https://github.com/noureddineme/tamazight-layout) - Linux tamazight keyboard layout (bouth Tifinagh and Latin)
 - [Deniable Encrypted Keys Database](https://github.com/abderraouf-adjal/dekdb-format-specifications) - Deniable Encrypted Keys Database (DEKDB) format specifications
+- [awesome-open-minds](https://github.com/open-minds/awesome-openminds-team) awesome useful links, resources and shiny things for CS students, geeks and developers
```
Comment (youknowriad) [inline README.md:38]: Could you move this to the top of the "other" category (to keep an alphabetical order).

**Review by youknowriad — CHANGES_REQUESTED** [review]
Thank's for the PR. These Repositories are awesome!. I like the name "Tchamba", Genius! :) 

Though make sure to reorder `awesome-open-minds` and rebase your branch.

**Comment by Fcmam5** [comment]
thank you @IOAyman, @youknowriad . Done

**Comment by IOAyman** [comment]
Only a step more...
```bash
git remote add upstream https://github.com/youknowriad/awesome-algeria.git && git rebase upstream/master
```

**Comment by Fcmam5** [comment]
I did that, and I still seeing that there's a conflict

**Comment by youknowriad** [comment]
@Fcmam5 `git fetch upstream` before the rebase

**Comment by IOAyman** [comment]
`git fetch upstream` before rebasing then

**Comment by youknowriad** [comment]
There's no conflict now, but you did not resolve the conflicts properly :) Make sure to correct your PR changes :)

**Comment by Fcmam5** [comment]
Sorry I messed it up, fixed now. Thank you

**Review by IOAyman — APPROVED** [review]

**Review by youknowriad — APPROVED** [review]

**Comment by youknowriad** [comment]
Thanks for the contribution @Fcmam5 feel free to join https://github.com/youknowriad/awesome-algeria/issues/7

**Pull request merged**